onyo/main.py

```diff
@@ -236,6 +236,66 @@ def setup_parser():
     )
     cmd_fsck.set_defaults(run=commands.fsck)
     #
+    # subcommand "get"
+    #
+    cmd_get = subcmds.add_parser(
+        'get',
+        description=textwrap.dedent(commands.get.__doc__),
+        formatter_class=SubcommandHelpFormatter,
+        help='print the requested key(s) for matching assets'
+    )
+    cmd_get.set_defaults(run=commands.get)
+    cmd_get.add_argument(
+        '-d', '--depth',
+        metavar='DEPTH',
+        type=int,
+        required=False,
+        default=0,
+        help='descend at most N levels of directories below the starting-point'
+    )
+    cmd_get.add_argument(
+        '-f', '--filter',
+        metavar='FILTER',
+        nargs='*',
+        type=str,
+        default='',
+        help='filter the results to match one or more conditions'
+    )
+    cmd_get.add_argument(
+        '-H', '--machine-readable',
+        dest='machine_readable',
+        action='store_true',
+        help='print output in a machine-readable format'
+    )
+    cmd_get.add_argument(
+        '-k', '--keys',
+        metavar='KEYS',
+        nargs='*',
+        default=['type', 'make', 'model', 'serial'],
+        help='keys to return'
```

Comment:
As per discussion in matrix, we now use `key(s)` to indicate the possibility of singular or multiple entries. I have changed this to be more explicit: `'asset-key value(s) to return'`

Reply:
The way I view it, pseudo-keys are asset keys too. They're just a special set of asset keys, because they're present in the name of the asset rather than in the contents of the asset.

Reply:
That is true in the abstract way, but adding that word here feels for me like it trys to do something explicitly that would not be true without it. If it just means all keys, pseudo or not, I would prefer to just use the word ` key value(s)`, alone for the sake of brevity then  :)

Reply:
I have adjusted this to `key value(s) to return`